### PR TITLE
Move AcousticBrainz Labs to MetaBrainz infra

### PR DESCRIPTION
**Pull request by paramsingh**
hey!

I added a docker image, an nginx conf, hosted the downloads folder on ftp.acousticbrainz.org and changed links accordingly.

### CODE REVIEW

File: Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM nginx:1.13.5
+
+RUN apt-get update && apt-get install -y --no-install-recommends jekyll node-less
```
Comment (alastair) [inline Dockerfile.prod]: ` && rm -rf /var/lib/apt/lists/*`

File: Dockerfile.prod [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN mkdir /code
```
Comment (alastair) [inline Dockerfile.prod]: `/site`, perhaps? :)

File: push.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# and push it to the Docker Hub, with an optional tag (by default "beta").
+#
+# Usage:
+#   $ ./push.sh  [tag]
+#
+# Examples:
+#   $ ./push.sh beta             # will push image with tag beta 
+#   $ ./push.sh v-2018-07-14.0   # will push images with tag v-2018-07-14.0 
+
+git describe --tags --dirty --always > .git-version
```
Comment (alastair) [inline push.sh]: Is this a metabraniz thing? I'm happy to keep this line in the file if it makes things consistent, but perhaps we should also add it in .gitignore?

File: Dockerfile.prod [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM nginx:1.13.5
```
Comment (alastair) [inline Dockerfile.prod]: The lab.acousticbrainz.org website is only the subdirectory - other directories here are for other projects/experiments.
We could either move Dockerfile & push.sh down a level, or break the website into its own repo. What do you think?

File: labs.acousticbrainz.org/_config.yml [hunk 5]
```diff
@@ -7,4 +7,4 @@ url: "http://acousticbrainz.org"
```
Comment (alastair) [inline labs.acousticbrainz.org/_config.yml:7]: let's fix http -> https in this file while we're at it.

File: labs.acousticbrainz.org/_config.yml [hunk 6]
```diff
@@ -7,4 +7,4 @@ url: "http://acousticbrainz.org"
 
 # Build settings
 markdown: kramdown
-permalink: /:title
+permalink: /:title.html
```
Comment (alastair) [inline labs.acousticbrainz.org/_config.yml]: it looks like what was missing here is a trailing `/` to make it correctly build the directory structure:

    permalink: /:title/

I'd prefer that we keep with this directory structure as it's the same as what we've historically had

**Review by alastair — CHANGES_REQUESTED** [review]
looks good, thanks!

File: Dockerfile.prod (hunk 4 above)
Comment (alastair) [reply Dockerfile.prod]: This is quite an old version of nginx, any reason?

**Comment by paramsingh** [comment]
@alastair done!

File: Dockerfile.prod (hunk 4 above)
Comment (paramsingh) [reply Dockerfile.prod]: Pushed down a level, old version was cargo culting :see_no_evil:

File: push.sh (hunk 3 above)
Comment (paramsingh) [reply push.sh]: Yeah, we use git tags to keep track in LB and AB and CB. Added to gitignore.

File: labs.acousticbrainz.org/_config.yml (hunk 6 above)
Comment (paramsingh) [reply labs.acousticbrainz.org/_config.yml]: done.

**Review by alastair — APPROVED** [review]

File: labs.acousticbrainz.org/_posts/2016-08-08-lastfm-genre-annotations.markdown [hunk 7]
```diff
@@ -58,19 +58,19 @@ MBIDs contain annotations in all three datasets).
 
 #### Last.fm beets mapping
 
-For details of these files, read our [paper and description](/ismir2016).
+For details of these files, read our [paper and description](/ismir2016.html).
```
Comment (alastair) [inline labs.acousticbrainz.org/_posts/2016-08-08-lastfm-genre-annotations.markdown:61]: I restored these to the original urls

**Comment by alastair** [comment]
thanks!

**Pull request merged**